src/utils/components/NetworkInterfaceModal/components/hooks/useNadsData.ts

```diff
@@ -18,11 +18,13 @@ const useNadsData: UseNadsData = (namespace) => {
       namespace: namespace,
     },
     //global ns to get usable nads
-    default: {
-      groupVersionKind: NetworkAttachmentDefinitionModelGroupVersionKind,
-      isList: true,
-      namespace: 'default',
-    },
+    ...(namespace !== 'default' && {
```

Comment:
can you please use DEFAULT_NAMESPACE const from `@kubevirt-utils/constants/constants`?
does it also make sense to check if this namespace is either `openshift-sriov-network-operator` or `openshift-multus`?
and can you also make consts for other ns used here?

Reply:
its not 1 of them, they are both global and need to be checked.
made them all consts